app/helpers/people_helper.rb

```diff
@@ -27,4 +27,17 @@ def person_as_badge(person, opt = nil)
       end
     end
   end
+
+  def person_time_clock_as_badge(person, hours, time_clock_period = nil)
+    content = pluralize(hours, "hour") + " " + i("hourglass-half") + " " + (time_clock_period ? time_clock_period.identifier_name : "No period")
+    if time_clock_period
+      link_to time_clock_period_path(time_clock_period, q: { person_id_eq: person.id }), class: "undecorated", "data-turbo": false do
```

Comment:
Since there's no person id search field on the `TimeClockPeriod#index` view, there's nothing besides the URL that tells you that you're filtered to only look at a particular person's punches... not sure if there's a nicer, relatively straightforward way to handle that other than using #135 to choose your person though.

Reply:
I've tried using `person_identifier_name_cont`, which does indicate on the search field, however, it doesn't correctly filter the person. If two people have similar `idenifier_name`s, it will show the clock in for both of them.